cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/exception/CloudStorageConfigurationFailedException.java

```diff
@@ -0,0 +1,19 @@
+package com.sequenceiq.cloudbreak.cm.exception;
+
+import java.util.List;
+
+import com.sequenceiq.cloudbreak.cm.ClouderaManagerOperationFailedException;
+
+public class CloudStorageConfigurationFailedException extends ClouderaManagerOperationFailedException {
+
+    private final List<String> errorReasons;
+
+    public CloudStorageConfigurationFailedException(String message, List<String> errorReasons) {
+        super(message);
+        this.errorReasons = errorReasons;
+    }
+
+    public List<String> getErrorReasons() {
+        return errorReasons;
+    }
```

Comment:
I cannot see this being queried or printed anywhere.

Reply:
Right this is not required anymore